registration-processor/core-processor/registration-processor-abis/pom.xml

```diff
@@ -11,7 +11,7 @@
         <version>1.1.5.5</version>
     </parent>
     <artifactId>registration-processor-abis</artifactId>
-    <version>1.1.5.5</version>
+    <version>1.1.5.5-p1-SNAPSHOT</version>
```

Comment:
check whether this is deployable project with Mono

Reply:
its  a service